.travis.yml

```diff
@@ -1,28 +1,27 @@
 language: go
 go:
-  - 1.11
-  - 1.9
+  - '1.9'
+  - '1.10'
+  - '1.11'
+  - '1.12'
+  - '1.13'
   - tip
 
-go_import_path: github.com/samuel/go-zookeeper
+go_import_path: github.com/z-division/go-zookeeper
 
 jdk:
   - oraclejdk9
 
 sudo: false
 
-branches:
-  only:
-    - master
-
 before_install:
   - make setup ZK_VERSION=${zk_version}
 
 before_script:
   - make lint
 
 script:
-  - jdk_switcher use oraclejdk9
+  - jdk_switcher use oraclejdk9 || true
```

Comment:
Educate me - is this a bash script? If so, are you intentionally swallowing an error? A comment would help.

Reply:
Dropped this change, didn't really have a whole lot of insight about it. (Just cherry picking from the upstream fork)

Reply:
> Just a quick question on some of the .travis.yml changes, since I don't think our team ever uses that.

Would you consider adding a CI pipeline for this fork?

Reply:
Sure, I'd consider adding a public CI pipeline.